recipes/liblsl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+# Conda build recipe for liblsl
+
+{% set name = "liblsl" %}
+{% set version = "1.14.1b8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sccn/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 196d28608f0c91fb589fd6392e5f4bf2c4d6d5ee4a68ae1df22bf7352a7621b4
+
+{% set common_cmake_args = "-DCMAKE_BUILD_TYPE=Release -DLSL_BUNDLED_PUGIXML=OFF -DLSL_UNIXFOLDERS=ON -S . -B build" %}
+
+build:
+  number: 0
+  script:
```

Comment:
Please put these into `bld.bat` and `build.sh` scripts and install on Windows into `LIBRARY_PREFIX`.